src/settings-list.js

```diff
@@ -171,8 +171,8 @@ module.exports = [
         toggle: function(value) {
             if (value === true) {
                 $('body').on('dblclick', '.chat-line', function() {
-                    chat.helpers.translate($(this).find('.message'), $(this).data('sender'), $(this).find('.message').data('raw'));
-                    $(this).find('.message').text('Translating..');
+                    chat.helpers.translate($(this).find('.message'), $(this).data('sender'), $(this).find('.message').text());
```

Comment:
For translation specifically, honestly I think `.text()` which filters out emotes is better, as you don't really want to give emotes to the translation. So it's more of a feature that emotes are being filtered, as they could throw off google translate.

So I think this PR is good as it is. If not, to fix the bug, you'd need to add `decodeURIComponent()` around the message as right now I'm pretty sure the data is being encoded twice (resulting in the bug).